### PR TITLE
✨ Add cognativex analytics to amp

### PR DESCRIPTION
**Pull request by m7mdmakkeh**
See issue #38456 

Implementing CognativeX tracking analystics (pageviews & heartbeats) to AMP
This is a test version as we would like to explore more possibilities later after this first integration with AMP!

### CODE REVIEW

**Comment by m7mdmakkeh** [comment]
There are some dead links extensions/amp-analytics/analytics-vendors-list.md and it's causing a 
Those links are not ours, they were there before,, those are the links:

https://my.lotame.com/t/83h37h9/overview-1st-party-data-collection-in-google-amp (404)
http://www.nielsen.com/us/en/solutions/capabilities/nielsen-marketing-cloud.html (404)
 https://manual.sensorsdata.cn/sa/latest/web-js-sdk-7548149.html (0)
 
 What should we do?
 @dmanek

File: extensions/amp-analytics/analytics-vendors-list.md [hunk 1]
```diff
@@ -333,7 +339,7 @@ Adds support for Linkpulse. Configuration details can be found at [docs.linkpuls
 
 Type attribute value: `lotame`
 
-Adds support for Lotame. More information and configuration details can be found at [my.lotame.com](https://my.lotame.com/t/83h37h9/overview-1st-party-data-collection-in-google-amp).
+Adds support for Lotame. More information and configuration details can be found at `https://my.lotame.com/t/83h37h9/overview-1st-party-data-collection-in-google-amp`.
```
Comment (erwinmombay) [inline extensions/amp-analytics/analytics-vendors-list.md:342]: why were these links changed?

Comment (m7mdmakkeh) [reply extensions/amp-analytics/analytics-vendors-list.md:342]: Those are dead links(404) and where blocking the PR. I addressed the issue in the comment above but haven't been getting any help. Any direction on how to proceed with the issues? Can you please check the above comment for additional details on the Percy test too..

**Comment by kshehadex** [comment]
@newmuis

**Comment by kshehadex** [comment]
@newmuis can you check our case please?

File: extensions/amp-analytics/analytics-vendors-list.md (hunk 1 above)
Comment (m7mdmakkeh) [reply extensions/amp-analytics/analytics-vendors-list.md:342]: @erwinmombay

**Pull request closed without merging**